extra/language/rust.nix

```diff
@@ -30,8 +46,32 @@ with lib;
       # Used by tools like rust-analyzer
       name = "RUST_SRC_PATH";
       value = toString cfg.packageSet.rustPlatform.rustLibSrc;
-    }];
+    }]
+    ++
+    (lib.optionals hasLibraries [
```

Comment:
Do we actually want to make the separation into libraries/includes? Would it not be easier to have just one package options that adds both?

Reply:
Since includes were for C and specifically C_INCLUDE_PATH, I don't see how it's necessary to port that to the rust options.